django/db/backends/sqlite3/introspection.py

```diff
@@ -217,9 +217,15 @@ def get_primary_key_column(self, cursor, table_name):
         fields_sql = create_sql[create_sql.index('(') + 1:create_sql.rindex(')')]
         for field_desc in fields_sql.split(','):
             field_desc = field_desc.strip()
-            m = re.match(r'(?:(?:["`\[])(.*)(?:["`\]])|(\w+)).*PRIMARY KEY.*', field_desc)
-            if m:
-                return m[1] if m[1] else m[2]
+            method_1_match = re.match(r'PRIMARY KEY\((.+?)[\,)]', field_desc)
+            if method_1_match:
+                if method_1_match:
+                    return method_1_match[1]
+                else:
+                    return None
+            method_2_match = re.match(r'(?:(?:["`\[])(.*)(?:["`\]])|(\w+)).*PRIMARY KEY.*', field_desc)
+            if method_2_match:
+                return method_2_match[1] if method_2_match[1] else method_2_match[2]
```

Comment:
Could we possibly switch all of this regex parsing logic to use `sqlparse` instead?